### ISSUE TITLE
RuntimeWarning: invalid value encountered in cast

### ISSUE THREAD

**Issue by theroggy** [mattijn/topojson#201]
Dedup gives a RuntimeWarning in some cases because in those cases one of the values in array_bk_sarcs is nan.

Script to reproduce:
```
import geopandas as gpd
import topojson

url = "https://github.com/geofileops/geofileops/raw/main/tests/data/polygon-parcel.gpkg"
gdf = gpd.read_file(url, driver="GPKG")
topo = topojson.Topology(gdf, topoquantize=False)
print("Ready")
```

Warning:
```
/home/runner/micromamba-root/envs/test/lib/python3.10/site-packages/topojson/core/dedup.py:107: RuntimeWarning: invalid value encountered in cast
    data["bookkeeping_shared_arcs"] = array_bk_sarcs.astype(np.int64).tolist()
```

**Comment by mattijn** [comment]
Thanks for reporting the issue. Currently a few days out and only available on mobile. Will come back soon to this! Thanks again for reporting!

**Comment by olofevenvik** [comment]
I got the same issue!
and this guy as well: https://github.com/topojson/topojson/issues/342

**Comment by mattijn** [comment]
Fixed by #202
